src/app/components/PortraitVideoCarousel/index.tsx

```diff
@@ -13,7 +13,10 @@ import { BumpLoader } from '../MediaLoader';
 import PortraitVideoPromo from './PortraitVideoPromo';
 import PortraitCarouselNavigation from './PortraitVideoCarouselNavigation';
 
-export const PROMO_ITEM_WIDTH = 180;
+export const PROMO_ITEM_WIDTH_MIN = 147;
+export const PROMO_ITEM_WIDTH_GROUP_3_MIN = 156;
+export const PROMO_ITEM_WIDTH_GROUP_4_MIN = 201;
+export const PROMO_ITEM_WIDTH_GROUP_5_MIN = 197;
```

Comment:
I am not sure if this is the best place for these values - it feels like they should belong in a styles file. Perhaps the top level styles file? src/app/components/PortraitVideoCarousel/index.styles.ts?